### PR TITLE
embeddings: remove TEI embeddings client. 

### PR DESCRIPTION
**Pull request by pattonjp**
followup per #352 

### PR Checklist

- [x] Read the [Contributing documentation](https://github.com/tmc/langchaingo/blob/main/CONTRIBUTING.md).
- [x] Read the [Code of conduct documentation](https://github.com/tmc/langchaingo/blob/main/CODE_OF_CONDUCT.md).
- [x] Name your Pull Request title clearly, concisely, and prefixed with the name of the primarily affected package you changed according to [Good commit messages](https://go.dev/doc/contribute#commit_messages) (such as `memory: add interfaces for X, Y` or `util: add whizzbang helpers`).
- [x] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `Fixes #123`).
- [x] Describes the source of new concepts.
- [x] References existing implementations as appropriate.
- [x] Contains test coverage for new functions.
- [x] Passes all [`golangci-lint`](https://golangci-lint.run/) checks.

### CODE REVIEW

**Comment by pattonjp** [comment]
examples ci build is failing due to new dependency.

File: vectorstores/milvus/milvus.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ import (
 type Store struct {
 	dropOld          bool
 	async            bool
-	initialized      bool
```
Comment (eliben) [inline vectorstores/milvus/milvus.go:21]: The changes in this file seem unrelated to the main goal of the PR, or am I missing something? It'd be good to keep the PR for a single purpose only

File: llms/tei/text_embeddings_inference..go [hunk 2]
```diff
@@ -0,0 +1,43 @@
+package tei
```
Comment (eliben) [inline llms/tei/text_embeddings_inference..go:1]: Maybe it's worth adding a comment here explaining this package? Specifically, it's not a usual LangChainGo LLM, but only a shell for invoking an embeddings API.

**Review by eliben — COMMENTED** [review]
Thanks!

**Comment by eliben** [comment]
@pattonjp friendly ping - do you intend to continue with this PR?

**Comment by pattonjp** [comment]
after the embeddings factor, it appears that the tei embeddings client is no longer needed. i am now able to use openai llm as client to my local tei instance.

File: chroma_vectorstore_example
Comment (eliben) [inline chroma_vectorstore_example:1]: This binary file was removed recently - seems like some sort of rebase artifact? Can you make sure it's not added back?

**Review by eliben — CHANGES_REQUESTED** [review]

**Comment by eliben** [comment]
> @eliben after the embeddings factor, it appears that the tei embeddings client is no longer needed. i am now able to use openai llm as client to my local tei instance.

That's a great outcome :)

**Review by tmc — APPROVED** [review]
LGTM

**Pull request merged**